src/main/java/duke/exceptions/EmptyCommandException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exceptions;
+
+public class EmptyCommandException extends Exception{
```

Comment:
This code viiolates the coding standards. Consider adding spaces between `Exception` and `{`.